### PR TITLE
feat(minor-voting-verifier)!: add migrate endpoint to update gateway address

### PR DESCRIPTION
**Pull request by eguajardo**
## Description

https://axelarnetwork.atlassian.net/browse/AXE-3938

### CODE REVIEW

**Comment by cjcobb23** [comment]
This is just updating a value in the config right? If so, this pattern is different than what we have been doing in some other places. For instance, to update the voting threshold, we just defined an execute msg that does so, and restrict calling it to governance (https://github.com/axelarnetwork/axelar-amplifier/blob/3e23d98c9facb0184ac60602c729f0b32ce8d3bc/contracts/voting-verifier/src/msg.rs#L65)

I actually prefer using the migration handler instead of defining an execute msg, but we should converge on one style for updating the config params.

File: contracts/voting-verifier/src/contract.rs [hunk 1]
```diff
@@ -75,6 +75,15 @@ pub fn query(deps: Deps, _env: Env, msg: QueryMsg) -> StdResult<Binary> {
     }
 }
 
+#[cfg_attr(not(feature = "library"), entry_point)]
+pub fn migrate(
+    deps: DepsMut,
+    _env: Env,
+    msg: MigrateMsg,
+) -> Result<Response, axelar_wasm_std::ContractError> {
+    migrate::set_source_gateway_address(deps, msg.source_gateway_address)
```
[the remark this answers is not in the document]
Comment (eguajardo) [reply contracts/voting-verifier/src/contract.rs:84]: I believe there is a different task for versioning all contracts: https://axelarnetwork.atlassian.net/browse/AXE-3817

[the remark this answers is not in the document]
Comment (haiyizxx) [reply contracts/voting-verifier/src/contract.rs:84]: we left that out during last release due to time constraint. need to migrate per contract

[the remark this answers is not in the document]
Comment (cjcobb23) [reply contracts/voting-verifier/src/contract.rs:91]: should we set this in instantiation as well?

[the remark this answers is not in the document]
Comment (cjcobb23) [reply contracts/voting-verifier/src/contract.rs:91]: yeah I think we should set the contract version in instantiation as well

**Review by cjcobb23 — APPROVED** [review]

**Pull request merged**